src/pages/List/List.js

```diff
@@ -1,7 +1,75 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import CardList from './Card/CardList';
+import Filter from './Filter/Filter';
+import styled from 'styled-components';
 
 const List = () => {
-  return <div>List</div>;
+  const [personCount, setPersonCount] = useState(0);
+  const [cardList, setCardList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/listFilter.json')
```

Comment:
```suggestion
    fetch('data/listFilter.json')
```
localhost:3000은 빼도 됩니다.